### PR TITLE
Added the "our products" part to company page

### PR DESCRIPTION
**Pull request by NjaalSoerland**
Now displays a list of all the products we offer to companies (ITEX, bedpress etc.) with dynamic icons to match

### CODE REVIEW

File: src/components/icons/UtlysningIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
```
Comment (kharann) [inline src/components/icons/UtlysningIcon.tsx:1]: remove all react imports

File: src/components/views/CompanyView/OurProducts.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import { css } from "@theme";
+import { Flex } from "@components/primitives";
+import BedpressIcon from "src/components/icons/BedpressIcon";
+import ItexIcon from "src/components/icons/ItexIcon";
+import OfflineIcon from "src/components/icons/OfflineIcon";
+import UtlysningIcon from "src/components/icons/UtlysningIcon";
+import TecktalksIcon from "src/components/icons/TecktalksIcon";
+import Text from "@components/atoms/Text";
+
+const OurProducts = () => {
```
Comment (kharann) [inline src/components/views/CompanyView/OurProducts.tsx:11]: repeating code

File: src/components/views/CompanyView/index.tsx [hunk 3]
```diff
@@ -31,6 +32,7 @@ export const CompanyView: VFC<CompanyViewProps> = (props: CompanyViewProps) => {
         </a>
       </Box>
       <Product blocks={product.content} />
+      <OurProducts></OurProducts>
```
Comment (kharann) [inline src/components/views/CompanyView/index.tsx:35]: <OurProducts/>

**Review by kharann — CHANGES_REQUESTED** [review]

**Comment by NjaalSoerland** [comment]
Still some repeating code in OurProducts.tsx but i believe it's more effort than it's worth to rewrite it to be more compact. The component is never going to be reused so i think it's fine as is:)

**Pull request merged**